.jenkins/pipelines/standalone/dotnet-p1-tests.Jenkinsfile

```diff
@@ -25,6 +25,7 @@ pipeline {
         TEST_TYPE =         "dotnet"
         LCOV_INFO =         "lcov-${GIT_COMMIT[0..7]}-${TEST_TYPE}.info"
         PACKAGE_INSTALL =   "${UBUNTU_VERSION == '20.04' ? 'Ubuntu-2004' : 'Ubuntu-1804'}_${PACKAGE_NAME}.${PACKAGE_EXTENSION}"
+        // MYST_RELEASE =      "1"
```

Comment:
Did this pipeline have any issues with the release flag turned on?

Reply:
pytorch also seems to have this commented out.

Reply:
Yes, they both has issues so I have kept the comment to indicate that its something we need to enable